src/include/ipc/info.h

```diff
@@ -51,6 +65,7 @@ struct sof_ipc_fw_version {
 	uint8_t time[10];
 	uint8_t tag[6];
 	uint32_t abi_version;
+	struct sof_ipc_cc_version compiler;
```

Comment:
This will modify the size of the structure, not it's not backwards compatible.

Reply:
This could go in FW boot extended metadata to preserve fw_version struct..

Reply:
So it should be realized similar to  sof_ipc_dma_buffer_data or sof_ipc_window (starts with sof_ipc_ext_data_hdr and send via mailbox in platform_boot_complete function)? Or sof_ipc_cc_version should be part of sof_man_fw_header or sof_man_module_manifest?

Reply:
Extended metadata is a minor ABI update, so yes starts with sof_ipc_ext_data_hdr

Reply:
done